src/effects/VST3/VST3Effect.h

```diff
@@ -38,17 +39,72 @@ namespace Steinberg
 
 class ParameterChangesProvider;
 
+
+struct VST3EffectSettings
+{
+   std::unordered_map<Steinberg::Vst::ParamID, Steinberg::Vst::ParamValue> mValues;
+};
+
+struct VST3Wrapper
+{
+   // For the time being, here we have members that are needed
+   // to iterate parameters and extract preset state
+   //
+   Steinberg::IPtr<Steinberg::Vst::IEditController> mEditController;
+   Steinberg::IPtr<Steinberg::Vst::IComponent>      mEffectComponent;
+   Steinberg::IPtr<internal::ComponentHandler>      mComponentHandler;
```

Comment:
See my comments here: https://github.com/audacity/audacity/issues/2606

I determined that the uses of component handler do not belong in the fetching and storing of settings.  Instead, these are (some of?) the steps that need to move into `TransferDataToWindow` and ultimately to the Validator.

So I think the wrapper should only have the first two of these handles.